source/shared/cpp/ObjectModel/FeatureRegistration.h

```diff
@@ -3,11 +3,28 @@
 #pragma once
 
 #include "pch.h"
+#include "ParseUtil.h"
 
 namespace AdaptiveSharedNamespace
 {
     constexpr const char* const c_adaptiveCardsFeature = "adaptiveCards";
 
+    struct KeyHash
```

Comment:
>KeyHash [](start = 11, length = 7)

`KeyHash` and `KeyEquals` should probably be named `CaseInsensitiveKeyHash` and `CaseInsensitiveKeyEquals`, respectively

Reply:
(otherwise, looks great!)

---
In reply to: [322983007](https://github.com/microsoft/AdaptiveCards/pull/3434#discussion_r322983007) [](ancestors = 322983007)